src/pybind/mgr/nfs/tests/test_nfs.py

```diff
@@ -802,6 +821,48 @@ def _do_test_create_export_rgw(self, cluster_id, expected_exports):
         assert export.clients[0].addresses == ["192.168.0.0/16"]
         assert export.cluster_id == cluster_id
 
+    def test_create_and_remove_export(self):
+        with self._mock_orchestrator(True), \
+                mock.patch('nfs.module.Module.get_osdmap') as get_osdmap:
+            get_osdmap.return_value.dump.return_value.get.return_value = {}
+            self._reset_temp_store(empty=True)
+            m = Module('nfs', '', '')
+            assert m._cmd_nfs_cluster_create(
+                cluster_id='foo'
+            ) == (0, '', 'foo cluster already exists')
+            assert self.temp_store == {'foo': {'conf-nfs.foo': TestNFS.RObject('conf-nfs.foo', '')}}
+
+            assert m._cmd_rgw_export_create_rgw(
+                bucket='bucket',
+                cluster_id='foo',
+                pseudo_path="/mybucket",
+            )[0] == 0
+            ex1 = dedent("""
+            EXPORT {
+                FSAL {
+                    name = "RGW";
+                    user_id = "nfs.foo.bucket";
+                    access_key_id = "the_access_key";
+                    secret_access_key = "the_secret_key";
+                }
+                export_id = 1;
+                path = "bucket";
+                pseudo = "/mybucket";
+                access_type = "RW";
+                squash = "none";
+                attr_expiration_time = 0;
+                security_label = true;
+                protocols = 4;
+                transports = "TCP";
+            }
+            """).lstrip()
+            assert self.temp_store == {
+                'foo': {
+                    'conf-nfs.foo': TestNFS.RObject('conf-nfs.foo', '%url "rados://nfs-ganesha/foo/export-1"\n\n'),
+                    'export-1': TestNFS.RObject('export-1', ex1),
+                }
+            }
```

Comment:
With this patch we want verify empty common config object creation, right ? Then why is export creation being tested?
Where is export deletion being done ? Let's split it in two different tests or please amend the commit message to reflect the actual change